src/main/java/com/exadel/sandbox/em_notif/service/impl/EmployeeNotificationServiceImpl.java

```diff
@@ -0,0 +1,87 @@
+package com.exadel.sandbox.em_notif.service.impl;
+
+import com.exadel.sandbox.em_notif.dto.EmployeeNotificationBaseDto;
+import com.exadel.sandbox.em_notif.dto.EmployeeNotificationResponseDto;
+import com.exadel.sandbox.em_notif.entity.EmployeeNotification;
+import com.exadel.sandbox.em_notif.repository.EmplyeeNotificationRepository;
+import com.exadel.sandbox.em_notif.service.EmployeeNotificationService;
+import com.exadel.sandbox.exception.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+
+@Component
+@RequiredArgsConstructor
+@Transactional
+public class EmployeeNotificationServiceImpl implements EmployeeNotificationService {
+
+    private final EmplyeeNotificationRepository emplyeeNotificationRepository;
+    private final ModelMapper mapper;
+
+
+    @Override
+    public List<EmployeeNotificationResponseDto> getEmployeeNotifications() {
+
+        List<EmployeeNotification> employeeNotificationList = emplyeeNotificationRepository.findAll();
+
+        List<EmployeeNotificationResponseDto> employeeNotificationResponseDtos = new ArrayList<>();
+
+        for (EmployeeNotification emplNotif:
+             employeeNotificationList) {
+            employeeNotificationResponseDtos.add(fullMap(emplNotif));
+        }
+
```

Comment:
Please  write this method something like that: 
List<EmployeeNotification> employeeNotificationList = emplyeeNotificationRepository.findAll();

        return employeeNotificationList.stream()
                .map(this::fullMap)
                .collect(Collectors.toList());

Reply:
Also please check other places and try to use stream where possible